huxley/api/views/delegate.py

```diff
@@ -5,20 +5,38 @@
 from rest_framework.authentication import SessionAuthentication
 from rest_framework.response import Response
 
-from huxley.api.permissions import IsSchoolDelegateAdvisorOrSuperuser, IsPostOrSuperuserOnly
+from huxley.api.mixins import ListUpdateModelMixin
+from huxley.api.permissions import IsSchoolDelegateAdvisorOrSuperuser, IsPostOrSuperuserOnly, IsChairOrSuperuser
 from huxley.api.serializers import DelegateSerializer
 from huxley.core.models import Delegate
 
 
 class DelegateList(generics.CreateAPIView):
-    authentication_classes = (SessionAuthentication,)
+    authentication_classes = (SessionAuthentication, )
     queryset = Delegate.objects.all()
-    permission_classes = (IsPostOrSuperuserOnly,)
+    permission_classes = (IsPostOrSuperuserOnly, )
     serializer_class = DelegateSerializer
 
 
 class DelegateDetail(generics.RetrieveUpdateDestroyAPIView):
-    authentication_classes = (SessionAuthentication,)
+    authentication_classes = (SessionAuthentication, )
     queryset = Delegate.objects.all()
-    permission_classes = (IsSchoolDelegateAdvisorOrSuperuser,)
+    permission_classes = (IsSchoolDelegateAdvisorOrSuperuser, )
     serializer_class = DelegateSerializer
+
+
+class DelegateCommitteeDetail(generics.ListAPIView, ListUpdateModelMixin):
+    authentication_classes = (SessionAuthentication, )
+    queryset = Delegate.objects.all()
+    serializer_class = DelegateSerializer
+    permission_classes = (IsChairOrSuperuser, )
+
+    def get_queryset(self):
```

Comment:
As mentioned in the other PR, we can actually just override this method in the DelegateList view to either return `Delegate.objects.all()`, or filtered by the committee ID if that parameter is provided. Then, the URI would become `/api/delegates?committee_id=...`